lib/src/components/NxThreatCounter/NxThreatCounter.scss

```diff
@@ -54,16 +54,17 @@
 }
 
 .nx-threat-counter-container {
-  display: inline-grid;
+  display: inline-flex;
```

Comment:
Didn't the designers agree that it should be fully inline, for instance that here I should be seeing the first two counters or so on the same line as the asdfasfasdf text?
![image](https://user-images.githubusercontent.com/3630091/180455073-b9e1ea98-f0d4-4443-8a7d-7a911470c721.png)

Reply:
> Didn't the designers agree that it should be fully inline, for instance that here I should be seeing the first two counters or so on the same line as the asdfasfasdf text? ![image](https://user-images.githubusercontent.com/3630091/180455073-b9e1ea98-f0d4-4443-8a7d-7a911470c721.png)

Hi Ross, after consulting with @JasonDSwearingen and after discussions, they decided that we're changing this to flex: f9f1673b5705fee581e328107afbc544230bbf16

They're working on updating Kaleido and they said they'll approve this ticket on Monday after the update is done 👍